decent-dao/fractal-interface
# Proposal eth value

## Description
This PR adds an ETH value input field to the create proposal flow so that ETH can be sent with DAO transactions


## Notes



## Issue / Notion doc (if applicable)
https://github.com/decent-dao/fractal-interface/issues/863


## Testing
1) Browse to a DAO you can create proposals on
2) Create a TX / proposal with a non-zero ETH value
3) Verify on the Gnosis UI that the TX will send the specified amount of ETH


## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/84364476/210448630-fccb7bfe-03b7-4f84-a0b9-afc47c08f064.png)
